exasol/ds/sandbox/lib/dss_docker/create_image.py

```diff
@@ -0,0 +1,99 @@
+import docker
+import logging
+from datetime import datetime
+from docker.types import Mount
+from exasol.ds.sandbox.lib import pretty_print
+from importlib_metadata import version
+from pathlib import Path
+
+from exasol.ds.sandbox.lib.config import ConfigObject, SLC_VERSION
+# renaming proposal:
```

Comment:
Please create a ticket, that will be a shotgun change and should go into its own PR. And, remove the comment afterwards.